Add Progressive logo and rework app footer

brandonhundt opened the pull request:
- Adds Progressive Logo. Progressive requires 300px x 100px, so I had to rework the other logos to be consistent (Tinku request).
- slightly nudges up the font size in the footer to match.
- If Progressive goes away, we can revert this as I can't say I like this more, but it works for now.

## To Review

- [ ] Use the Preview link located in the Now comment below.

> ...or...

- [ ] Checkout Branch.
- [ ] Run `yarn`.
- [ ] Run `yarn dev:start`.
- [ ] Go to [localhost:3000](https://localhost:3000).

> ...then...

- [ ] View footer. check responsive

brandonhundt committed:
feat(appFooter): add progressive logo reworks logos

brandonhundt committed:
remove accidental commit

rpeterman-gp commented on components/AppFooter/AppFooter.styles.ts:
```diff
@@ -19,13 +19,14 @@ export const appFooterStyles = makeStyles()((theme) => ({
-    width: '100%',
-    maxWidth: theme.typography.pxToRem(250),
+    width: '300px',
+    maxWidth: theme.typography.pxToRem(300),
```
The image width in this case should remain 100% so it fills the containers grid columns.

rpeterman-gp commented on components/AppFooter/AppFooter.styles.ts:
```diff
@@ -71,30 +75,46 @@ export const appFooterStyles = makeStyles()((theme) => ({
   producedByMuiOl: {
-    justifyContent: 'center'
+    display: 'grid',
+    gridTemplateColumns: `${theme.typography.pxToRem(300)}`,
```
Since the requirement is a fixed 300px, don't use rem's since those will scale with users font size settings. This would apply to any column with values in other containers.

rpeterman-gp replied:
Unfortunately, though Progressive wants 300px width, we may need to display it slightly smaller on mobile screens. When padding is taken into account, forcing 300px width could break page layouts and cause horizontal scrolling and/or content clipping. Columns should probably use `minmax(300px, 1fr)`.

rpeterman-gp commented on components/AppFooter/AppFooter.styles.ts:
```diff
@@ -71,30 +75,46 @@ export const appFooterStyles = makeStyles()((theme) => ({
+    gridGap: theme.typography.pxToRem(12),
+    justifyContent: 'center',
+    [theme.breakpoints.up(370)]: {
+      gridTemplateColumns: `repeat(2, ${theme.typography.pxToRem(300)})`
```
The math for this doesn't work. shouldn't be going to 2 columns until viewport is > 600px (plus container inline padding and grid gap)

brandonhundt committed:
feat(appFooter): rework for progressive

rpeterman-gp committed:
fix(appfooter): logo list item styles

- add inline padding to ensure Ford Foundation logo doesn't touch edges
- set line height to 0 to ensure logos are vertically centered

rpeterman-gp reviewed (approved):
Looks good. I made a couple tweaks to address vertical alignment and padding issues (Ford Foundation logo). :shipit:
